docs/quick_start.md

```diff
@@ -0,0 +1,46 @@
+# Quick Start
+
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Run tests](#run-tests)
+- [Write tests](#write-tests)
+- [FAQ and Troubleshooting](#faq-and-troubleshooting)
+
+LISA can be used to run test against the local node, or a remote node; if it is
+used to run against a remote node, you don't need to configure anything on the
+remote node.
+
+![deploy](img/deploy.svg)
+
+## Prerequisites
+
+LISA can be launched on a Windows or a Linux OS. It needs to be installed on a
+computer which has network access to the platform and the node to be tested. 
+
+- It is recommended that this computer at least has 2 CPU cores and 4GB memory.
+
+## Installation
+
+Follow the installation guide below to install LISA on your OS.
+
+- [Installation on Linux](installation_linux.md)
+- [Installation on Windows](installation_windows.md)
+
+## Run tests
+
+Follow [Getting started with Azure](run_test/quick_run.md) to quickly run your
+first test. See [Run LISA](run_test/run.md) for more detailed information.
+
+## Write tests
+
+Follow [How to write a test case]() to quickly have your first own test. See
+[Info for developers](write_test/write_case.md) for more detailed information.
+
+- [How-to](write_test/write_case.md)
```

Comment:
"How-to write tests"?

Reply:
Will elaborate on this when structuring write_test part.